src/videojs.ads.js

```diff
@@ -211,7 +224,10 @@ var
       // safari requires a call to `load` to pick up a changed source
       player.load();
       // and then resume from the snapshots time once the original src has loaded
+      // in some browsers (firefox) `canplay` may not fire correctly.
+      // Reace the `canplay` event with a timeout.
       player.one('contentcanplay', tryToResume);
+      player.ads.tryToResumeTimeout = player.setTimeout(tryToResume, 2000);
```

Comment:
Probably should suffix this with an underscore so it's obvious it's an internal variable.